Anderson Promoters/views/Libraries and Composites.csv

```diff
@@ -11,9 +11,9 @@ Interlab negative control,Transformation of a basic plasmid,,True,pSB1C3,,R0040,
 Libraries and Composite Parts,,Click on column headers for more information as to what to put into the column,,,,,,,,,,,,,,,,,,,,,,,
 "Short and human friendly name for the part, e.g., ""GFP Reporter""",Comments about design decisions. E.g. GC content or restriction sites removed,A description of how the part is meant to function and any information about the part that is not to do with how it was designed.,"True if this is a final product to be produced, false if this is an intermediate stage being specified for compactness","If the composite is being placed into a plasmid, provide the name of the plasmid. If the composite is being transformed into the genome, provide the name of the locus for incorporation.","Identity and strand constraints on parts, expressed as Part X RELATION Part Y, where RELATION is ""same as"", ""different from"", ""same orientation as"", ""different orientation from""","Part names (left to right: 5'->3') should match the names of other basic or composite parts. Add more columns as needed. Indicate alternatives with commas, RC(name) for reverse complement",,,,,,,,,,,,,,,,,,,
 Part/Library Name,Design Notes,Part Description,Final Product,Backbone/locus,Constraints,Part 1,Part 2,Part 3,Part 4,Part 5,Part 6,Part 7,Part 8,Part 9,,,,,,,,,,,
-Anderson Promoters in vector,,,True,pSB1C5,,D2001,D1001,"J23100, J23101, J23102, J23103, J23104, J23105, J23106, J23107, J23108, J23109, J23110, J23111, J23113, J23114, J23115, J23116, J23117, J23118, J23119",D1002,D2002,,,,,,,,,,,,,,,
-,,,False,,,,,,,,,,,,,,,,,,,,,,
-,,,False,,,,,,,,,,,,,,,,,,,,,,
+Anderson Promoters in vector,,,True,pSB1C5,,D2001,D1001,"J23100, J23102, J23105, J23109, J23110, J23114, J23116, J23119",D1002,D2002,,,,,,,,,,,,,,,
+Anderson Promoters in pSB1C3,,,True,pSB1C3,,D2001,D1001,"J23101, J23104, J23106, J23107, J23111, J23113, J23115, J23117, J23118",D1002,D2002,,,,,,,,,,,,,,,

```

Comment:
I see that J23108 and J23103 aren't there in pSB1C3 - were they unable to be built? Your commit comment says "some" and says J23103 is one of them, but not what the others were.
It would also be good to have notes added in the description field of "unable to build"